app/src/ui/lib/filter-list.tsx

```diff
@@ -198,7 +199,10 @@ export class FilterList<T extends IFilterListItem> extends React.Component<
   public constructor(props: IFilterListProps<T>) {
     super(props)
 
-    this.state = createStateUpdate(props)
+    this.state = {
+      ...createStateUpdate(props),
+      filterValue: props.filterText || '',
+    }
```

Comment:
Is this needed at all? 🤔 It looks like `createStateUpdate` already sets that `filterValue` (and makes it lowercase)
```suggestion
    this.state = createStateUpdate(props),
```

Reply:
Absolutely! Thanks